include/gatekeeper_config.h

```diff
@@ -67,6 +73,31 @@ struct dynamic_config {
 
 	/* The Lua file for initializing dynamic configuration. */
 	char             *dynamic_config_file;
+
+	struct mailbox   mb;
+	/*
+	 * The callee that finished processing the return message
+	 * needs to increment this counter, so that the Dynamic config block
+	 * can finish its operation.
+	 */
+	rte_atomic16_t   num_returned_instances;
+};
+
+/* Define the possible command operations for Dynamic config block. */
+enum dy_cmd_op {
+	GT_UPDATE_POLICY_RETURN,
+};
+
+/* Currently, the GT blocks are the only writers of Dynamic config mailbox. */
+struct dy_cmd_entry {
+	enum dy_cmd_op  op;
+
+	union {
+		/* GT_UPDATE_POLICY_RETURN */
+		struct {
+			char return_msg[RETURN_MSG_MAX_LEN];
```

Comment:
Adding field `unsigned int gt_lcore` here would be a cheap but helpful piece of information to add to the log entry at `process_dy_cmd()`. The information would be helpful for diagnosis and debugging, not for processing.